babel_util/parsers/aminer.py

```diff
@@ -123,10 +123,13 @@ def parse_line(self, line):
     import argparse
     import sys
     parser = argparse.ArgumentParser()
+    # adds first argument as input file
```

Comment:
No new code added here, and the comments arent substantive. Either expand comments (and follow Google style guide) or remove this file from the change set.